pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -695,6 +696,22 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 			}
 
 			// Set replicas
+			// if the configuration use only one replica we increment
+			if strings.HasPrefix(string(cr.Spec.Environment), constants.RhdmPrefix) && serverSet.JbpmCluster.EnableCluster != nil && *serverSet.JbpmCluster.EnableCluster {
```

Comment:
Why is there checked if it is rhdm kie server? jbpm cluster should work for both types of Kie Server, right? And mainly as it's called jbpm cluster, it mainly make sense, that this configuration is available in rhpam

Reply:
ok